imports/client/components/PuzzlePage.tsx

```diff
@@ -1177,24 +1177,26 @@ class PuzzlePage extends React.Component<PuzzlePageProps, PuzzlePageState> {
       <DocumentTitle title={`${activePuzzle.title} :: Jolly Roger`}>
         {this.state.isDesktop ? (
           <div className="puzzle-page">
-            <SplitPanePlus
-              split="vertical"
-              defaultSize={DefaultSidebarWidth}
-              minSize={MinimumSidebarWidth}
-              pane1Style={{ maxWidth: MaximumSidebarWidth }}
-              autoCollapse1={-1}
-              autoCollapse2={-1}
-            >
-              {sidebar}
-              <PuzzlePageContent
-                puzzle={activePuzzle}
-                allTags={this.props.allTags}
-                guesses={this.props.allGuesses}
-                displayNames={this.props.displayNames}
-                document={this.props.document}
-                isDesktop={this.state.isDesktop}
-              />
-            </SplitPanePlus>
+            <div className="puzzle-page-inner">
```

Comment:
If there's a reason for the inner div to stay, in the name of consistency we could add it to the narrow view as well, move the .narrow to the new inner div, and update the .puzzle-page.narrow style to .puzzle-page-inner.narrow. But it isn't necessary.

Reply:
I thought we needed a container with `position: fixed` so that we could then put the `position: absolute` container inside of it, but that doesn't actually seem to be necessary, so we can drop it.